assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss

```diff
@@ -20,11 +20,11 @@
 
 	.googlesitekit-km-widget-tile {
 		display: flex;
+		flex-direction: column;
 		justify-content: space-between;
-		padding: $grid-gap-phone;
+		padding: 12px $grid-gap-phone;
```

Comment:
Even though this wasn't a part of the ACs, I've updated the horizontal padding here (on smaller viewports) to match the Figma designs.